README.md

```diff
@@ -31,63 +42,153 @@ envs:
 - A_SECRET_PARAM_TWO: the value for secret two
 ```
 
-## How to create your own step
+## Simple usage
 
-1. Create a new git repository for your step (**don't fork** the *step template*, create a *new* repository)
-2. Copy the [step template](https://github.com/bitrise-steplib/step-template) files into your repository
-3. Fill the `step.sh` with your functionality
-4. Wire out your inputs to `step.yml` (`inputs` section)
-5. Fill out the other parts of the `step.yml` too
-6. Provide test values for the inputs in the `bitrise.yml`
-7. Run your step with `bitrise run test` - if it works, you're ready
+### Example using public IDs
 
-__For Step development guidelines & best practices__ check this documentation: [https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md).
+<!-- TODO: change git urls to step references after we publish it -->
```

Comment:
Also add that to the corresponding card (the definition of done can just say something like "fix all TODOs in READMEs")